roles/openshift_logging_curator/tasks/main.yaml

```diff
@@ -12,6 +12,13 @@
     openshift_logging_curator_image_prefix: "{{ openshift_logging_curator_image_prefix | default(__openshift_logging_curator_image_prefix) }}"
     openshift_logging_curator_image_version: "{{ openshift_logging_curator_image_version | default(__openshift_logging_curator_image_version) }}"
 
+- name: Detect running upgrade
+  set_fact:
+    legacy_config_exists: "{{ openshift_logging_facts['curator']['configmaps']['logging-curator']['config.yaml'] is defined }}"
+
+- debug:
```

Comment:
todo: eventually remove
also as a fyi you can do `debug: var=legacy_config_exists` and it would provide you with the same output :)